File: kafis-practices-backend/Practice.BLL/Models/Admin/AdminOrderResponse.cs
using Practice.Application.DTOs.User.Student;
using Practice.Domain.Core.Common.Enums;
using System;
using System.Collections.Generic;

namespace Practice.Application.Models.Admin
{
    public class AdminOrderResponse
    {
        public GradeLevelEnum GradeLevel { get; set; }
        public string Specialty { get; set; }
        public string Specialization { get; set; }
        public DateTime? StartDate { get; set; }
        public DateTime? EndDate { get; set; }

        public IEnumerable<StudentOrderDTO> StudentOrders { get; set; }
    }
}

File: kafis-practices-frontend/libs/interfaces/src/lib/admin-order-response.ts
import { GradeLevel } from "@practice/enums";

export interface AdminOrderResponse {
  year: number;
  gradeLevel: GradeLevel;
  specialty: string;
  specialization: string;
  startDate: string;
  endDate: string;
  studentOrders: {
    organizationName: string;
    students: {
      studentName: string;
      teacherPosition: string;
      teacherName: string;
    }[];
  }[];
}
